apps/docs.blocksense.network/app/docs/contracts/overview/page.mdx

```diff
@@ -163,15 +79,13 @@ When an update is posted to the contract, an event is emitted - “DataFeedsUpda
 
 See the [Integration guide](./integration-guide/using-data-feeds/aggregated-data-feed-store) for more information.
 
-Limitations: one can read a whole feed, i.e. all of its historical data, but cannot read more than one feed in a call. If a feed is of a greater stride (anything but stride 0), one can pass start slot and slots count (optional) parameters to read a specific data slice. For example, we have a data feed of stride 5 (i.e. 1024b or 32 slots) and we want to read only the last 3 slots, we would pass 29 (counting starts from 0) as start slot and 3 as slots count.
-
 ### Access Control
 
-This contract is called when writing to the ADFS storage. Its role is to check if the writer (caller of the transaction) has permission to write to the storage; if not - reverts the transaction.
+This contract ensures the caller has the necessary permissions to perform specific actions.
 
 ### CL Adapters
 
-  The `cl-adapters` folder contains Chainlink aggregator adapter contract (`CLAggregatorAdapter.sol`) and a feed registry contract (`/registries/CLFeedRegistryAdapter.sol`). The CL aggregator adapter contract implements the Chainlink aggregator interface, enabling it to interact with the `UpgradeableProxyADFS` contract to fetch stored data. The idea is that the `UpgradeableProxyADFS` holds the storage in itself and only uses the `AggegatedDataFeedStore` contract functions to know how to read or write to its own storage.
+The `cl-adapters` folder contains Chainlink aggregator adapter contract (`CLAggregatorAdapter.sol`) and a feed registry contract (`/registries/CLFeedRegistryAdapter.sol`). The CL aggregator adapter contract implements the Chainlink aggregator interface, enabling it to interact with the `ADFS` contract to fetch stored data. The feed registry contract is responsible for mapping asset addresses to their corresponding Chainlink data feeds. It provides a mechanism for users to retrieve the address of a specific data feed based on the asset address.
```

Comment:
ditto: Link to https://github.com/blocksense-network/blocksense/tree/main/libs/ts/contracts/contracts/cl-adapters